pkg/inspector/image-inspector.go

```diff
@@ -203,12 +230,12 @@ func aggregateBytesAndReport(bytesChan chan int) {
 		select {
 		case bytes, open := <-bytesChan:
 			if !open {
-				log.Printf("Finished Downloading Image (%dKb downloaded)", bytesDownloaded/1024)
+				log.Printf("DEBUG: Image download finished (%dKb downloaded)", bytesDownloaded/1024)
```

Comment:
@simon3z Should these be debug messages? if they are, then we shouldn't print this always.